agent/tags.go

```diff
@@ -95,9 +108,28 @@ func (t *tagFetcher) Fetch(l logger.Logger, conf FetchTagsConfig) []string {
 		}
 	}
 
+	// Attempt to add the EC2 meta-data fetched from user-default path suffixes
+	if len(conf.TagsFromEC2MetaDataPaths) > 0 {
+		paths, err := parseTagValuePathPairs(conf.TagsFromEC2MetaDataPaths)
+		if err != nil {
+			l.Error(fmt.Sprintf("Error parsing meta-data tag and path pairs: %s", err.Error()))
```

Comment:
This will show an error but continue, is that the intention?

Reply:
Good catch — nope, this should ‼️💥

Reply:
Actually — now I think about it a little more I’m actually not sure about this. I think it’s fine to throw an error but continue and it’s how it already works...